hooks/after_setup/add_host_storage_domain.yml

```diff
@@ -33,7 +33,7 @@
   with_items: "{{ gluster.hosts }}"
   loop_control:
     loop_var: host
-  when: "gluster is defined and 'hosts' in gluster"
+  when: gluster is defined and "hosts in gluster" and gluster.hosts | length > 1
```

Comment:
why do you need the double quotes around "hosts in gluster"?

Reply:
Removed